lib/content-services/src/lib/document-list/components/library-status-column/library-status-column.component.spec.ts

```diff
@@ -16,9 +16,54 @@
  */
 
 import { LibraryStatusColumnComponent } from './library-status-column.component';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ContentTestingModule } from '../../../testing/content.testing.module';
+import { CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA } from '@angular/core';
+import { Site } from '@alfresco/js-api';
 
 describe('LibraryStatusColumnComponent', () => {
-  it('should be defined', () => {
-    expect(LibraryStatusColumnComponent).toBeDefined();
-  });
+    let fixture: ComponentFixture<LibraryStatusColumnComponent>;
+    let component: LibraryStatusColumnComponent;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ContentTestingModule, LibraryStatusColumnComponent],
+            schemas: [CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA]
```

Comment:
Do you need NO_ERRORS_SCHEMA too? Ideally if we could avoid that because based on https://angular.dev/api/core/NO_ERRORS_SCHEMA it's discouraged